moped-editor/src/views/projects/newProjectView/NewProjectMapBaseMap.js

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import { Icon, makeStyles, Typography } from "@material-ui/core";
+import { SpeedDial, SpeedDialAction } from "@material-ui/lab";
+
+export const useStyles = makeStyles(theme => ({
+  speedDialStreets: {
+    color: "black",
+    backgroundImage: "url(/moped/static/images/mapStreets.jpg) !important",
+  },
+  speedDialAerial: {
+    color: "white",
+    backgroundImage: "url(/moped/static/images/mapAerial.jpg) !important",
+  },
+  mapStyleToggle: {
+    position: "absolute",
+    top: "-10.45rem",
+    right: "-2.5rem;",
+  },
+  mapStyleToggleImage: {
+    width: "5rem",
+    borderRadius: ".5rem",
+  },
+  mapStyleToggleLabel: {
+    position: "relative",
+    bottom: "-1.3rem",
+    fontSize: ".8rem",
+    fontWeight: "bold",
+  },
+  mapStyleToggleLabelIcon: {
+    position: "relative",
+    bottom: "-1.5rem",
+    fontSize: ".8rem",
+    fontWeight: "bold",
+    marginRight: "4px",
+  },
+  mapStyleToggleTypography: {
+    position: "absolute",
+    height: "100%",
+    width: "100%",
+    top: "0",
+  },
+  mapStyleToggleTypographyOpen: {
+    backgroundColor: "rgba(0,0,0,.25)",
+    position: "absolute",
+    height: "100%",
+    width: "100%",
+    top: "0",
+  },
+}));
```

Comment:
Extensive styles for detailed changes. Basically it controls the speed-dial for map style switching.